include/exadg/operators/navier_stokes_calculators.cpp

```diff
@@ -23,6 +23,66 @@
 
 namespace ExaDG
 {
+template<int dim, typename Number>
+DivergenceCalculator<dim, Number>::DivergenceCalculator()
+  : matrix_free(nullptr), dof_index_vector(1), dof_index_scalar(2), quad_index(0)
+{
```

Comment:
this was just copied

Reply:
what about the values 1 and 2 here? The shear rate calculator uses 0 always for initialization.

Reply:
I changed all of these to 0. They were set in the constructor anyways ... but now its uniformly initialized to 0.